src/main/kotlin/net/maizegenetics/phgv2/cli/Phg.kt

```diff
@@ -18,5 +18,5 @@ class Phg : CliktCommand() {
 }
 
 fun main(args: Array<String>) = Phg()
-    .subcommands(Initdb(), CreateRanges(), AlignAssemblies(), CreateRefVcf(), BuildMafVcf(), LoadVcf())
+    .subcommands(Initdb(), AgcCompress(), CreateRanges(), AlignAssemblies(), CreateRefVcf(), BuildMafVcf(), LoadVcf())
```

Comment:
Is this in the order the command is used?

Reply:
I added AgcCompress() arbitrarily.  Compressing your fastas into agc could occur in different places.  Perhaps this happens after ALignAssemblies(), but since the AGC compression doesn't change the fastas, it doesn't actually matter.  I can rearrange the order if you prefer something different.

Reply:
I think it would be good if the help showed the commands in the order they are used.

Reply:
Where do you think AgcCompress() should go?  After AlignAssemblies()?

Reply:
I don't know. Seems like a utility outside the usual workflow?

Reply:
As mentioned in slack, I think this needs to go after alignAssemblies and before the VCF commands.

Reply:
order has been changed as per slack.